connector/extensions/dcp-impl/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    `java-library`
+    id("application")
+    id("jacoco")
+}
+
+tasks.test {
+    useJUnitPlatform() // Ensures JUnit 5 is used
+    finalizedBy(tasks.jacocoTestReport)
+}
+
+tasks.jacocoTestReport {
+    dependsOn(tasks.test)
+    reports {
+        xml.required.set(true)
+        html.required.set(true)
+    }
+}
+
+repositories {
+    mavenCentral()
+}
```

Comment:
all of this common configuration could stay in the main build gradle file in the `allprojects` block, so all the projects will inherit the same config